python/setup.py.in

```diff
@@ -23,6 +23,21 @@ with open('@PADDLE_SOURCE_DIR@/python/requirements.txt') as f:
 if '${CMAKE_SYSTEM_PROCESSOR}' not in ['arm', 'armv7-a', 'aarch64']:
     setup_requires+=["opencv-python"]
 
+# the prefix is sys.prefix which should always be usr
+paddle_bin_dir = 'local/opt/paddle/bin'
```

Comment:
Tested in docker which will install under `/usr/local/local/opt/paddle/bin/paddle_usage` if python's main path is under `/usr/local`, should this be `opt/paddle/bin`?